app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -99,25 +99,25 @@ The salary you must be paid to get a visa varies according to  whether you are e
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £36,745
-* outer London, you must earn £34,514
-* London fringe, you must earn £31,350
-* the rest of England, you must earn £30,000
+* inner London, you must earn $salaries_skilledworkerteacherfulltimemin_innerlondon$
```

Comment:
@Sarah-DfE I think the idea was that we would move these variables to the nonuk variables file

This means we will need to:

Before  https://github.com/DFE-Digital/get-into-teaching-app/pull/4035 is merged to production:
* take out these changes from https://github.com/DFE-Digital/get-into-teaching-app/pull/4035
* take out these variables from the nonuk config file in https://github.com/DFE-Digital/get-into-teaching-app/pull/4035

After  https://github.com/DFE-Digital/get-into-teaching-app/pull/4035 is merged to production and after all other variable PRs are merged to production:
* add a change here to add these salary thresholds into the nonuk variables file
* update all the variable references in this PR to have `nonuk` rather than `salaries` at the front

I'll set this PR to draft for now - let's come back to it and work through it together!